### ISSUE TITLE
Allow later websockets release

### ISSUE THREAD

**Issue by fabaff** [sanic-org/sanic-testing#19]
`websockets` is [pinned](https://github.com/sanic-org/sanic-testing/blob/main/setup.py#L51). The latest `websockets` is 9.1 and this release is fixing a [authentication vulnerability](https://websockets.readthedocs.io/en/stable/changelog.html) which was introduced with 8.0.

**Comment by TheKevJames** [comment]
@ahopkins just a heads up that this will need to get a bump in [sanic](https://github.com/sanic-org/sanic/blob/44b657dc5934c7177cdc3d0debf0c4addb852a2a/setup.py#L91) as well -- currently, the latest release of `sanic-testing` is not compatible with any version of Sanic itself, since they rely on exclusive ranges of the `websockets` version.

For example, with poetry:
```
  SolverProblemError

  Because sanic-testing (0.6.0) depends on websockets (>=9.0)
   and sanic (21.3.4) depends on websockets (>=8.1,<9.0), sanic-testing (0.6.0) is incompatible with sanic (21.3.4).
  So, because SAMPLE_APP depends on both sanic (21.3.4) and sanic-testing (0.6.0), version solving failed.
```

**Comment by ChihweiLHBird** [comment]
This issue has been solved in d029c53e374b3c686e3513489d1dc7b9f65b1634.
